graylog2-web-interface/src/views/components/dashboard/BigDisplayModeConfiguration.jsx

```diff
@@ -47,50 +47,56 @@ const ConfigurationModal = ({ onSave, onCancel, view }: ConfigurationModalProps)
 
   return (
     <Modal show bsSize="large" onHide={onCancel}>
-      <Modal.Header closeButton>
-        <Modal.Title>Configuring Full Screen</Modal.Title>
-      </Modal.Header>
-      <Modal.Body>
-        <Input id="refresh-interval"
-               type="number"
-               name="refresh-interval"
-               label="Refresh Interval"
-               help="After how many seconds should the dashboard refresh?"
-               onChange={({ target: { value } }) => setRefreshInterval(Number.parseInt(value, 10))}
-               required
-               value={refreshInterval} />
-
-        <FormGroup>
-          <ControlLabel>Tabs</ControlLabel>
-          <ul>
-            {availableTabs.map(([idx, title]) => (
-              <li key={`${idx}-${title}`}>
-                <Checkbox inline
-                          checked={queryTabs.includes(idx)}
-                          onChange={event => (event.target.checked ? addQueryTab(idx) : removeQueryTab(idx))}>
-                  {title}
-                </Checkbox>
-              </li>
-            ))}
-          </ul>
-          <HelpBlock>
+      <form onSubmit={_onSave} data-testid="display-mode-config-form">
+        <Modal.Header closeButton>
+          <Modal.Title>Configuring Full Screen</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <Input id="refresh-interval"
+                 type="number"
+                 min="1"
+                 name="refresh-interval"
+                 label="Refresh Interval"
+                 help="After how many seconds should the dashboard refresh?"
+                 onChange={({ target: { value } }) => setRefreshInterval(value ? Number.parseInt(value, 10) : value)}
```

Comment:
When the value falsy is you set the refreshInterval to that? Why not to 0 or a other default value?